dev-libs/utfcpp/utfcpp-4.0.5.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="UTF-8 with C++ in a Portable Way"
+DESCRIPTION="C++ developers still miss an easy and portable way of handling Unicode encoded \
+strings. The original C++ standard (known as C++98 or C++03) is Unicode agnostic. Some progress \
+has been made in the later editions of the standard, but it is still hard to work with Unicode \
+using only the standard facilities.
+
+I came up with a small, C++98 compatible generic library in order to handle UTF-8 encoded strings. \
+For anybody used to work with STL algorithms and iterators, it should be easy and natural to use. \
+The code is freely available for any purpose - check out the [license](./LICENSE). The library has \
+been used a lot since the first release in 2006 both in commercial and open-source projects and \
+proved to be stable and useful."
+HOMEPAGE="https://github.com/nemtrif/utfcpp"
+COPYRIGHT="2006-2022 Nemanja Trifunovic"
+LICENSE="Boost v1.0"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="ffc668a310e77607d393f3c18b32715f223da1eac4c4d6e0579a11df8e6b59cf"
+PATCHES="utfcpp-$portVersion.patchset"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	utfcpp = $portVersion
+	devel:utfcpp = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. $cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE=Release
+	make -Cbuild $jobArgs
```

Comment:
Upstream says that the [CMake file is deprecated](https://github.com/nemtrif/utfcpp/blob/master/CMakeLists.txt) and instead [suggests just copying the header files directly](https://github.com/nemtrif/utfcpp/blob/master/README.md#installation).

Maybe this should just be removed and the header files installed directly with `cp`? Or are the installed CMake files needed downstream?

Reply:
Tried that, just cp the files to $includeDir/... taglib doesn't pick it up then.